index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="container-top">
+            <h1>Rita Daniel</h1>
+            <img src="./images/Union2x.svg" alt="menu-bar">
+        </div>
+        <div class="primary-text">
+            <h2>Hey There. I’m Rita</h2>
+            <h3>I’am a Software Developer</h3>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+            see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+            <div class="social-media-icon" social-media-icon>
+                <img src="./images/Social Media.png" alt="social-media-icon">
+            </div>
+        </div>
```

Comment:
- [ ] Kindly I recommend you to use `section` tag for headline section instead of `div` tag because It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.